### PR TITLE
Analysis name and type are now populated from submission archive name

### LINKED ISSUE #842: Make some strings automatically populated when user creates the submission
Steps:

1. Open https://qanode.arachnenetwork.com/
2. Submissions -> Create submission 
3. Add .zip file, example 
submission

### PR DESCRIPTION
**Pull request by ssuvorov-fls**
fixes #842

### CODE REVIEW

File: src/modules/Submissions/components/List/ModalСreateSubmission/container.js [hunk 1]
```diff
@@ -83,6 +85,16 @@ class ModalCreateSubmissionBuilder extends ContainerBuilder {
             options = files.map(({ name }) => name);
           } else {
             options = await getFileNamesFromZip(files[0]);
+            const fileName = files[0];
+            const parts = fileName.originalName.split("-");
```
Comment (dmitrys-odysseus) [inline src/modules/Submissions/components/List/ModalСreateSubmission/container.js:89]: If split yields 3 fragments, we lose the 3rd one. I suggest we limit the split by 2 parts.

File: src/modules/Submissions/components/List/ModalСreateSubmission/container.js [hunk 2]
```diff
@@ -83,6 +85,16 @@ class ModalCreateSubmissionBuilder extends ContainerBuilder {
...
+            let name = "";
+            let type = "";
+            if (parts.length >= 2) {
+              type = getTypeByShortPrefix(parts[0]);
+              name = parts[1];
+            } 
+            dispatchProps.setAnalysisType(type);
```
Comment (dmitrys-odysseus) [inline src/modules/Submissions/components/List/ModalСreateSubmission/container.js:96]: If the split fails to deliver 2 pieces, we set both to parts to "". That doesn't make much sense to me.
If we have a hard requirement to have type, fail fast.
If not - at least preserve the originalName in the 'name'

**Review by dmitrys-odysseus — CHANGES_REQUESTED** [review]

File: src/modules/Submissions/components/List/ModalСreateSubmission/container.js (hunk 2 above)
Comment (ssuvorov-fls) [reply src/modules/Submissions/components/List/ModalСreateSubmission/container.js]: It is not a hard requirement.
Done.

File: src/modules/Submissions/components/List/ModalСreateSubmission/container.js (hunk 1 above)
Comment (ssuvorov-fls) [reply src/modules/Submissions/components/List/ModalСreateSubmission/container.js:89]: Greater or equal sign in comparison with the length of the array is changed to equal sign.
Done.

**Review by dmitrys-odysseus — APPROVED** [review]

**Pull request merged**